src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int DATA_TO_CHECK = 0;
+    private static final int NAME = 1;
+    private static final int HOURS = 2;
+    private static final int INCOME_PER_HOUR = 3;
+    private static final String PATTERN_OF_DATE = "dd.MM.yyyy";
```

Comment:
```suggestion
    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

lets make DateTimeFormatter as consantr, not only date format